marketplace_builder/views/pages/tutorials/performance/cache.liquid

```diff
@@ -15,10 +15,17 @@ Page caching means you create a static HTML site, and serve it bypassing the who
 
 ```liquid
 {% raw %}
-cache_static_page_for: 0 
+# views/pages/home.liquid
```

Comment:
Please dont include file path inside code blocks, when someone copies it over it will throw syntax error.

`#### views/pages/home.liquid` before the code block should the trick

Reply:
Thank you! Fixed them.